gusto/solvers/linear_solvers.py

```diff
@@ -164,24 +181,28 @@ def __init__(self, equations, alpha=0.5,
                 logger.warning("default quadrature degree most likely not sufficient for this degree element")
             self.quadrature_degree = (5, 5)
 
-        super().__init__(equations, alpha, solver_parameters,
+        super().__init__(equations, alpha, tau_u, tau_r, tau_t, solver_parameters,
                          overwrite_solver_parameters)
 
     @timed_function("Gusto:SolverSetup")
     def _setup_solver(self):
 
         equations = self.equations
         dt = self.dt
-        beta_ = dt*self.alpha
```

Comment:
It looks to me like `alpha` can be passed in but is no longer used anywhere - is that correct? Maybe it makes sense to have `alpha` default to 0.5 and the various `taus default to `None` but then set them to `alpha` if they are not specified?

Reply:
Thanks Jemma, I have updated so that if not tau values are given, they default to alpha